### PR TITLE
Creation of post-it library

### PR DESCRIPTION
**Pull request by ferminrp**
Hey, just created a few post its that I think are usefull for design thinking =)

### CODE REVIEW

**Comment by ferminrp** [comment]
Also added a template for a gantt diagram

**Comment by anumithaapollo12** [comment]
Damn, love the idea of this library!

File: libraries.json [hunk 1]
```diff
@@ -626,5 +626,31 @@
...
+    "name": "Sticky Notes",
+    "description": "Post its in every color for your design thinking sessions.",
+    "authors": [
+      {
+        "name": "ferminrp",
+        "url": "https://github.com/ferminrp"
+      }
+    ],
+    "source": "ferminrp/post-it.excalidrawlib",
+    "preview": "braweria/post-it.png",
```
Comment (dwelle) [inline libraries.json:640]:
```suggestion
    "preview": "ferminrp/post-it.png",
```

File: libraries.json [hunk 2]
```diff
@@ -626,5 +626,31 @@
...
+    "name": "Gantt Diagram",
+    "description": "A Gantt diagram template you can use to plan your projects.",
+    "authors": [
+      {
+        "name": "ferminrp",
+        "url": "https://github.com/ferminrp"
+      }
+    ],
+    "source": "ferminrp/gantt.excalidrawlib",
+    "preview": "braweria/gantt.png",
```
Comment (dwelle) [inline libraries.json:653]:
```suggestion
    "preview": "ferminrp/gantt.png",
```

**Comment by dwelle** [comment]
How about making the folded corner into its own shape so it has white background?

https://excalidraw.com/?#json=6098649352241152,mESj5lWi8XsymrTwmdN3Hw

Otherwise you can't put the post-it notes above other items.

image

**Comment by ferminrp** [comment]
I just updated the post it library following @dwelle 's suggestion

https://github.com/excalidraw/excalidraw-libraries/pull/156/#issuecomment-865007096

File: libraries.json [hunk 3]
```diff
@@ -627,6 +627,32 @@
...
+    "description": "A Gantt diagram template you can use to plan your projects.",
+    "authors": [
+      {
+        "name": "ferminrp",
+        "url": "https://github.com/ferminrp"
+      }
+    ],
+    "source": "ferminrp/gantt.excalidrawlib",
+    "preview": "ferminrp/gantt.png",
+    "date": "2021-06-20"
```
Comment (dwelle) [inline libraries.json:654]:
```suggestion
    "date": "2021-06-22"
```

**Comment by dwelle** [comment]
Let's merge. Thanks!

**Pull request merged**